mojo/stdlib/stdlib/utils/variant.mojo

```diff
@@ -408,6 +417,26 @@ struct Variant[*Ts: Copyable & Movable](ImplicitlyCopyable, Movable):
                 return i
         return Self._sentinel
 
+    @staticmethod
+    fn _all_t_trivial[_dunder: String]() -> Bool:
+        var is_trivial = True
+
+        @parameter
+        for i in range(len(VariadicList(Ts))):
+
+            @parameter
+            if _dunder == "__del__":
+                is_trivial &= Bool(Ts[i].__del__is_trivial)
```

Comment:
IMO - its a bit easier to ready without the bit logic, and instead we should just return here
```suggestion
                if not(Bool(Ts[ i ].__del__is_trivial)):
                    return False
```